widget/form.go

```diff
@@ -128,16 +132,50 @@ func (f *Form) checkValidation(err error) {
 	f.submitButton.Enable()
 }
 
+func (f *Form) setUpHints(item *FormItem) fyne.CanvasObject {
+	_, ok := item.Widget.(fyne.Validatable)
+	if item.HintText == "" {
+		if !ok {
+			return item.Widget
+		}
+		if e, ok := item.Widget.(*Entry); ok && e.Validator == nil { // we don't have validation
+			return item.Widget
+		}
+	}
+
+	text := canvas.NewText(item.HintText, theme.PlaceHolderColor())
+	text.TextSize = theme.CaptionTextSize()
+	text.Move(fyne.NewPos(theme.Padding()*2, theme.Padding()*-0.5))
+	item.helperOutput = text
+	f.updateHelperText(item)
+	return fyne.NewContainerWithLayout(layout.NewVBoxLayout(), item.Widget, fyne.NewContainerWithoutLayout(text))
```

Comment:
Why does text need to be wrapped in a container without a layout?

Reply:
It was requested that it be moved closer to the element above than the standard padding amount. This was the easiest way to do it.

Reply:
So the core issue is that VBox adds padding between subelements? I'm confused how making one of the elements a container without a layout removes this padding

Reply:
Well, basically. All layouts add padding between elements. By using a container without layout I can control the position manually.

Reply:
So Box adds padding between item.Widget and the container, but then the container positions its child at (0,-padding)?

Reply:
yes

Reply:
The other approach is to write a custom layout which felt overkill for this 1-liner